splinepy/io/iges.py

```diff
@@ -35,4 +35,13 @@ def export(fname, splines):
     --------
     None
     """
+    # check if any spline contains volume elements
+    for ispl, spline in enumerate(splines):
+        if spline.para_dim > 2 or spline.dim > 3:
+            raise ValueError(
+                f"Volumetric {type(spline)} found at splines[{ispl}]"
+                "which cannot be handled by the .iges file format \n"
+                "Use a boundary representation instead by calling e.g."
+                "multipatch.boundary_multipatch()"
```

Comment:
```suggestion
                "Use a boundary representation instead by calling e.g."
                "spline.extract.boundaries()"
```
This is applicable for both single and multipatch